LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Christian Elies
```

Comment:
It would be nice if you could add my name here too ;)

Reply:
True 👍  Done.